Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -98,6 +98,17 @@ extension _RSA.Signing {
             }
         }
 
+        /// Construct an RSA public key with the specified parameters.
+        ///
+        /// Only the BoringSSL backend provides APIs to create a key from its parameters so we first create a BoringSSL
+        /// key, and then pass it to the platform-specific initializer that accepts a BoringSSL key.
+        ///
+        /// On Darwin platforms, this will serialize it to PEM format, and then construct a platform-specific key from
+        /// the PEM representation.
+        public init(n: some ContiguousBytes, e: some ContiguousBytes) throws {
```

Comment:
Generalised nit: `@inlinable` on generic methods, as far as possible. This feedback applies to any new generic function or type.

Reply:
FWICT there is a very deliberate use of `private` for the backing key types and `fileprivate` for  the type aliases which is why _existing_ generic initialisers are not currently inlinable either (e.g. the PEM and DER initialisers).

I think we'd probably need to take a wider view to achieve this so should we defer this to a future PR?

Reply:
Sounds good.